conf.ini

```diff
@@ -1,17 +1,17 @@
 # issuer information
-issuer_url = https://www.issuer.org
-issuer_email = contact@issuer.org
-issuer_name = University of Learning
-issuer_id = https://www.blockcerts.org/samples/2.0/issuer-testnet.json
-revocation_list=https://www.blockcerts.org/samples/2.0/revocation-list-testnet.json
+issuer_url = https://www.ust.hk/
```

Comment:
Can you exclude this file from the PR? `conf.ini` should be in .gitignore (sorry about that!); so if you wanted to make that change too, that would avoid this happening in the future.

Reply:
Hi Kim,

I just found that I wrongly submitted another commit which is on Jan 12. I will try withdraw that, I intended to just submit the one in Dec.

Thanks for your work!

Reply:
I will create another pull request =)